csrc/scheduler/transpose.cpp

```diff
@@ -714,29 +718,124 @@ std::shared_ptr<TransposeParams> getTransposeHeuristics(
       ceilDiv((int64_t)params->tile_size1 * (int64_t)params->tile_size2, 32l);
   max_unroll_factor = std::min(max_unroll_factor, max_unroll_factor_block);
 
-  // Compute maximum vectorize factor that can be used
-  size_t vectorize_factor1 = max_unroll_factor;
-  size_t vectorize_factor2 = max_unroll_factor;
+  // Note: [Computing Vectorization Width for Transpose]
```

Comment:
Can you please factor this part out as a function like `getTransposeVectorizationFactor`?